scipy/spatial/transform/rotation.py

```diff
@@ -1549,6 +1633,31 @@ def __getitem__(self, indexer):
         """
         return self.__class__(self._quat[indexer], normalized=True)
 
+    @classmethod
+    def identity(cls, num=None):
+        """Get identity rotation(s).
+
+        The identity rotation is its own inverse. Composition of a rotation
```

Comment:
I think the first sentence is not necessary. 

The second one is OK, but maybe it can be said simpler. The idea is that "Identity rotation = no rotation at all", don't know how to say it properly.

Reply:
It is hard to say :)
I went with `Composition with the identity rotation has no effect.`